### ISSUE TITLE
Should be better to load images that are “in view” and “visible”?

### ISSUE THREAD

**Issue by rmariuzzo** [Pentiado/angular-lazy-img#13]
I have the following scenario: a page with thousands of images distributed in tenths of nested `<li>`s. By default, these `<li>`s are hidden, but “in view” because its parent element (`<ul>`) is.

I think instead of only loading images that are “in view” we should also constraint to those that are “visible”.

What do you think?

**Comment by Pentiado** [comment]
Yeah, I agree. It totally makes sense. I'm overwhelmed with amount of tasks that I took at work right now but during Christmas break I will have some time to look at it.

**Comment by rmariuzzo** [comment]
Cool! I have made some changes on a separate branch, I will propose a PR for it.

**Comment by jup31** [comment]
Had you prupose your modifications on PR ?

**Comment by daangeerdink** [comment]
+1
